src/qml/ShakeToWag.qml

```diff
@@ -0,0 +1,181 @@
+                maxY = gyroscope_y_values[i];
+            }
+
+            if (gyroscope_z_values[i] <= minZ) {
+                minZ = gyroscope_z_values[i];
+            } else {
+                maxZ = gyroscope_z_values[i];
+            }
+        }
+
+        var amplitudeX = maxX - minX;
+        var amplitudeY = maxY - minY;
+        var amplitudeZ = maxZ - minZ;
+
+        minAmpl = Math.min(amplitudeX, amplitudeY, amplitudeZ);
+        maxAmpl = Math.max(amplitudeX, amplitudeY, amplitudeZ);
+
+        //temporarily commented out
+        //amplitudeGyroscopeData.text = "Amplitude gyroscope: x: " + Math.round(amplitudeX) + ", y: " + Math.round(amplitudeY) + ", z: " + Math.round(amplitudeZ);
+
+        if (maxAmpl - minAmpl > 700) {
+            return "Jump";
+        } else {
+            return "Shake";
+        }
+    }
+
+    ToggleButton {
+        id: goButton;
+        text: qsTr("GO");
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.horizontalCenter: parent.horizontalCenter
+        onClicked: function() {
+            isReadingMode = goButton.checked;
+            goButton.text = qsTr(isReadingMode ? "STOP": "GO");
+
+            if (isReadingMode) {
+                movementStatus.text = qsTr("Make a move");
+                gyroscope_x_values = [];
+                gyroscope_y_values = [];
+                gyroscope_z_values = [];
+            } else {
+                movementStatus.text = qsTr("Analyzing...");
+                var movement = recognizeMovement();
+                if (movement == "Jump") {
+                    BTConnectionManager.sendMessage("TAILER", []);
+                } else if (movement == "Shake") {
+                    BTConnectionManager.sendMessage("TAILS1", []);
+                }
+
+                movementStatus.text = "Your movement" + '\n' + "was recognized" + '\n' + "like a " + movement;
+            }
+        }
+    }
+
+    Column {
+        anchors.horizontalCenter: parent.horizontalCenter
+
+//temporarily commented out
+//        Text {
+//            id: gyroscopeData
+//            text: "Gyroscope: x: 0, y: 0, z: 0";
+//        }
+
+//        Text {
+//            id: amplitudeGyroscopeData
+//            text: "Amplitude Gyroscope: x: 0, y: 0, z: 0";
+//        }
+
+//        Text {
+//            id: vectorChangesData
+//            text: "Vector Changes: x: 0, y: 0, z: 0";
+//        }
+
+        Text {
+            id: movementStatus;
+            font.pointSize: 36;
+            anchors.horizontalCenter: parent.horizontalCenter
+        }
+
+//temporarily commented out
+//        Row {
+//            Text {
+//                text: qsTr("Action:")
+//            }
+
+//            ComboBox {
+//                id: action
+//                width: 150
+//                model: ["Jump", "Shake"]
+//            }
+//        }
+
+//        Row {
+//            Text {
+//                text: qsTr("Move:")
+//            }
+
+//            ComboBox {
```

Comment:
For this, just use a PickACommandSheet to select the command, that's what it's for (and what it's used for in the rest of the app) :)

Reply:
Thank you. I got it. I will do it

Reply:
Thanks. I've found this component in the code. I will use it in the future for similar cases instead of combobox. Please merge the PR.

Reply:
Great - it's commented out here anyway, so no problem :)